src/protocol/vm/state.rs

```diff
@@ -323,7 +323,7 @@ impl ProtocolSim for VMPoolState<PreCachedDB> {
         self.spot_prices
             .get(&(base.address, quote.address))
             .cloned()
-            .ok_or(SimulationError::NotFound("Spot prices".to_string()))
+            .ok_or(SimulationError::FatalError("Spot prices not found".to_string()))
```

Comment:
'Spot prices are empty'.
Also - is this a Fatal error? Can't the user call 'set_spot_prices' and then try again?

Reply:
Are we leaving it to the user to call `set_spot_prices`? Do they know about this? That was pretty unclear to me tbh - I thought this just happened when we delta transitioned.

Reply:
I think this is Fatal...